hawkular-inventory-rest-api/src/main/rest-doc/swagger.json

```diff
@@ -0,0 +1,1757 @@
+        "description" : "Environment can incorporate feeds and can contain resources and metrics."
+      } ]
+    },
+    "EnvironmentBlueprint" : {
+      "allOf" : [ {
+        "$ref" : "#/definitions/EntityBlueprint"
+      }, {
+        "type" : "object"
+      } ]
+    },
+    "EnvironmentUpdate" : {
+      "allOf" : [ {
+        "$ref" : "#/definitions/EntityUpdate"
+      }, {
+        "type" : "object"
+      } ]
+    },
+    "Feed" : {
+      "allOf" : [{
+        "$ref" : "#/definitions/IdentityHashedEntity"
+      }, {
+        "type" : "object",
+        "description" : "A feed represents a remote \"agent\" that is reporting its data to Hawkular."
+      }]
+    },
+    "FeedBlueprint" : {
+      "allOf" : [{
+        "$ref" : "#/definitions/EntityBlueprint"
+      }, {
+        "type" : "object"
+      }]
+    },
+    "FeedUpdate" : {
+      "allOf" : [{
+        "$ref" : "#/definitions/EntityUpdate"
+      }, {
+        "type" : "object"
+      }]
+    },
+    "IdentityHashTree" : {
+      "type" : "object",
+      "properties" : {
+        "path" : {
+          "$ref" : "#/definitions/RelativePath"
+        },
+        "hash" : {
+          "type" : "string"
+        },
+        "children" : {
+          "type" : "array",
+          "items" : {
+            "$ref" : "#/definitions/IdentityHashTree"
+          }
+        }
+      }
+    },
+    "IdentityHashedEntity" : {
+      "allOf" : [ {
+        "$ref" : "#/definitions/Entity"
+      }, {
+        "type" : "object",
+        "properties" : {
+          "identityHash" : {
+            "type" : "string"
+          }
+        },
+        "description" : "A super type of all entities that support identity hashing"
+      } ]
+    },
+    "JSON" : {
+      "type" : "object",
+      "description" : "Just a free form JSON."
+    },
+    "MetadataPack" : {
+      "allOf" : [{
+        "$ref" : "#/definitions/Entity"
+      }, {
+        "type" : "object",
+        "description" : "A metadata pack can incorporate global resource and metric types making them read-only."
+      }]
+    },
+    "MetadataPackBlueprint" : {
+      "allOf" : [{
+        "$ref" : "#/definitions/AbstractElementBlueprint"
+      }, {
+        "type" : "object",
+        "properties" : {
+          "members" : {
+            "type" : "array",
+            "uniqueItems" : true,
+            "items" : {
+              "$ref" : "#/definitions/CanonicalPath"
+            }
+          },
+          "name" : {
+            "type" : "string"
+          }
+        }
+      }]
+    },
+    "MetadataPackUpdate" : {
+      "allOf" : [{
+        "$ref" : "#/definitions/EntityUpdate"
+      }, {
+        "type" : "object"
+      }]
+    },
+    "Metric" : {
+      "allOf" : [ {
+        "$ref" : "#/definitions/IdentityHashedEntity"
+      }, {
+        "type" : "object",
+        "properties" : {
+          "type" : {
+            "$ref" : "#/definitions/MetricType"
+          },
+          "collectionInterval" : {
```

Comment:
It is, but it is defined on a metric, too. The idea being that the the one on the metric type is a "default" if not overridden by the metric.